nncf/tensorflow/quantization/algorithm.py

```diff
@@ -263,6 +264,9 @@ def _parse_range_init_params(self) -> TFRangeInitParams:
     def _parse_group_params(self, quant_config: Dict, quantizer_group: QuantizerGroup) -> None:
         group_name = quantizer_group.value
         params_dict = quant_config.get(group_name, {})
+        if not params_dict and self._target_device != 'VPU':
```

Comment:
The presets are not working for the VPU target. What do you think about adding some logs to inform users or raise an error that the VPU target does not support presets, currently?

Reply:
Done.